sql_generators/glean_usage/templates/baseline_clients_daily.view.sql

```diff
@@ -4,6 +4,12 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ daily_view }}`
 AS
 SELECT
-  *
+  *,
```

Comment:
This change is not part of the scope of the `active_users` view, could it be in a separate PR?